include/SFML/System/String.hpp

```diff
@@ -30,13 +30,24 @@
 ////////////////////////////////////////////////////////////
 #include <SFML/System/Export.hpp>
 #include <SFML/System/Utf.hpp>
+#include <cctype>
 #include <iterator>
 #include <locale>
 #include <string>
 
 
 namespace sf
 {
+namespace priv
+{
+    inline std::string toLower(std::string str)
+    {
+        for (char& c : str)
+            c = static_cast<char>(std::tolower(c));
```

Comment:
```suggestion
            c = static_cast<char>(std::tolower(static_cast<unsigned char>(c)));
```
See https://en.cppreference.com/w/cpp/string/byte/tolower#Notes

Reply:
Thanks. Fixed.